recipe/meta.yaml

```diff
@@ -20,7 +20,8 @@ source:
   url: https://www.neuron.yale.edu/ftp/neuron/versions/v{{ xy }}/{{ version }}/nrn-{{ version }}.tar.gz
   sha256: 85a0f999a9cdcc661b0066c32a78239e252f26e74eee5328bf1106393400bfc0
   patches:
-    - memacs.patch
+    - 0001-memacs.patch
+    - 0002-Update-deprecated-MPI_Adress-and-MPI_Type_struct.-Fix-if-a=b-warning.patch
```

Comment:
This is https://github.com/neuronsimulator/nrn/commit/58ce4c4966d2d3a2942761f69dd32e6d4272f385 which will be included in >=7.8.0.